src/behat/Api/Context/RestContextTrait.php

```diff
@@ -124,7 +124,27 @@ public function theCentreonApiDocumentation()
         $this->apiValidator = (new ValidatorBuilder())
             ->fromSchema($schema);
     }
-
+    
+    /**
+     * Initialize validator according to specific centreon web api documentation
+     *
+     * @Given the endpoints are described in Centreon Web API documentation version :version
+     */
+    public function theCentreonApiDocumentationVersion($version)
```

Comment:
why a new method ?
You could update the existing one with a regex : https://github.com/centreon/centreon-test-lib/blob/master/src/behat/Api/Context/ApiContext.php#L479